packaging/linux/Arch/PKGBUILD

```
@@ -69,43 +114,74 @@ prepare() {
 build() {
     export BRANCH="@GITHUB_BRANCH@"
     export BUILD_VERSION="@BUILD_VERSION@"
-    export COMMIT="@GITHUB_COMMIT@"
+    export COMMIT="${_commit}"
 
     export CC="gcc-${_gcc_version}"
     export CXX="g++-${_gcc_version}"
 
     export CFLAGS="${CFLAGS/-Werror=format-security/}"
     export CXXFLAGS="${CXXFLAGS/-Werror=format-security/}"
 
-    cmake \
-        -S "$pkgname" \
-        -B build \
-        -Wno-dev \
-        -D BUILD_DOCS=OFF \
-        -D BUILD_WERROR=ON \
-        -D CMAKE_INSTALL_PREFIX=/usr \
-        -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine \
-        -D SUNSHINE_ASSETS_DIR="share/sunshine" \
-        -D SUNSHINE_PUBLISHER_NAME='LizardByte' \
-        -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev' \
-        -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+    export CUDA_PATH=/opt/cuda
+    export NVCC_CCBIN="/usr/bin/g++-$_cuda_gcc_version"
+
+    export MAKEFLAGS="-j$(nproc)"
+
+    local _cmake_options=(
+      -S "$pkgname"
+      -B build
+      -Wno-dev
+      -D BUILD_DOCS=OFF
+      -D BUILD_WERROR=ON
+      -D CMAKE_INSTALL_PREFIX=/usr
+      -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine
+      -D SUNSHINE_ASSETS_DIR="share/sunshine"
+      -D SUNSHINE_PUBLISHER_NAME='LizardByte'
+      -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev'
+      -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+      -D SUNSHINE_ENABLE_CUDA=ON
+      -D SUNSHINE_ENABLE_DRM=ON
+      -D SUNSHINE_ENABLE_TRAY=ON
+      -D SUNSHINE_ENABLE_VAAPI=ON
+      -D SUNSHINE_ENABLE_WAYLAND=ON
+      -D SUNSHINE_ENABLE_X11=ON
+    )
+
+    if [[ "${_use_cuda::1}" == "t" ]]; then
+      _cmake_options+=(-DCUDA_FAIL_ON_MISSING=ON)
+    else
+      _cmake_options+=(-DCUDA_FAIL_ON_MISSING=OFF)
+    fi
+
+    if [[ "${_run_unit_tests::1}" == "t" ]]; then
+      _cmake_options+=(-DBUILD_TESTS=ON)
+    else
+      _cmake_options+=(-DBUILD_TESTS=OFF)
+    fi
+
+    cmake "${_cmake_options[@]}"
 
     appstreamcli validate "build/dev.lizardbyte.app.Sunshine.metainfo.xml"
     appstream-util validate "build/dev.lizardbyte.app.Sunshine.metainfo.xml"
     desktop-file-validate "build/dev.lizardbyte.app.Sunshine.desktop"
     desktop-file-validate "build/dev.lizardbyte.app.Sunshine.terminal.desktop"
 
-    make -C build
+    make "${MAKEFLAGS}" -C build
 }
 
 check() {
     export CC="gcc-${_gcc_version}"
     export CXX="g++-${_gcc_version}"
 
-    cd "${srcdir}/build/tests"
-    ./test_sunshine --gtest_color=yes
+    if [[ "${_run_unit_tests::1}" == "t" ]]; then
+      cd "${srcdir}/build/tests"
+      ./test_sunshine --gtest_color=yes
+    else
+      cd "${srcdir}/build"
+      ./sunshine --version
+    fi
```

```suggestion
    cd "${srcdir}/build"
    ./sunshine --version

    if [[ "${_run_unit_tests::1}" == "t" ]]; then
      cd "${srcdir}/build/tests"
      ./test_sunshine --gtest_color=yes
    else
```

This could run whether `_run_unit_tests` is enabled...

The exports can also go inside the if block.

---

Adjustments made